mod_cluster/extension/src/main/java/org/wildfly/extension/mod_cluster/ModClusterConfigAdd.java

```diff
@@ -35,6 +35,7 @@
     static final ModClusterConfigAdd INSTANCE = new ModClusterConfigAdd();
 
     private ModClusterConfigAdd() {
+        super(ModClusterConfigResourceDefinition.ATTRIBUTES);
```

Comment:
Yeah that one is strange, when I looked trough history it just looks some cruft from the past.